build.sbt

```diff
@@ -1,9 +1,9 @@
-lazy val scala212 = "2.12.10"
 lazy val scala211 = "2.11.12"
```

Comment:
let's remove 2.11 (and sbt 0.13, if there are any remnants). ScalaPB will drop support in the next release anyway: https://github.com/scalapb/ScalaPB/blob/master/CHANGELOG.md#0100

Reply:
Sounds good, changed.